packages/mux-player/src/template.ts

```diff
@@ -8,8 +8,16 @@ import { i18n, stylePropsToString } from './utils';
 import type { MuxTemplateProps } from './types';
 import { StreamTypes, toMuxVideoURL } from '@mux/playback-core';
 
+const getPropsCSS = (props: MuxTemplateProps) => {
+  const { tokens } = props;
+  if (!tokens.drm) return '';
```

Comment:
obviously if/when we add additional property-specific styles here, we wouldn't want a simple if + early bail. Didn't want to over-architect the guts until we needed it tho.